.rubocop_todo.yml

```diff
@@ -0,0 +1,62 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2018-03-03 10:25:38 -0500 using RuboCop version 0.52.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 11
+# Configuration parameters: CountComments, ExcludedMethods.
+Metrics/BlockLength:
+  Max: 68
+
+# Offense count: 14
+RSpec/AnyInstance:
+  Exclude:
+    - 'spec/api_caller_spec.rb'
+    - 'spec/api_object_spec.rb'
+    - 'spec/card_spec.rb'
+    - 'spec/transaction_spec.rb'
+    - 'spec/user_spec.rb'
+
+# Offense count: 1
+# Configuration parameters: Prefixes.
+# Prefixes: when, with, without
+RSpec/ContextWording:
+  Exclude:
+    - 'spec/api_caller_spec.rb'
+
+# Offense count: 2
+# Configuration parameters: Max.
+RSpec/ExampleLength:
+  Exclude:
+    - 'spec/transaction_spec.rb'
+
+# Offense count: 5
+# Configuration parameters: CustomTransform, IgnoreMethods.
+RSpec/FilePath:
```

Comment:
I opened https://github.com/financeit/marqeta/pull/6 to resolve this.

Reply:
So @andyw8 should we remove these exclusions since we'll be moving the files over?

Reply:
Let's merge this first, then I'll update #6.